pkg/kubernetes/module.go

```diff
@@ -381,12 +381,13 @@ func (d *DeleteKubeNodeModule) Init() {
 	d.Desc = "Delete kubernetes node"
 
 	drain := &task.RemoteTask{
-		Name:    "DrainNode",
-		Desc:    "Node safely evict all pods",
-		Hosts:   d.Runtime.GetHostsByRole(common.Master),
-		Prepare: new(common.OnlyFirstMaster),
-		Action:  new(DrainNode),
-		Retry:   5,
+		Name:        "DrainNode",
+		Desc:        "Node safely evict all pods",
+		Hosts:       d.Runtime.GetHostsByRole(common.Master),
+		Prepare:     new(common.OnlyFirstMaster),
+		Action:      new(DrainNode),
+		Retry:       2,
+		IgnoreError: true,
```

Comment:
The functionality of this `IgnoreError` field is not yet implemented. So if there are some errors, the result will not be what you expected.

Reply:
We want to ignore the error. Can we reserve it

Reply:
Just do not return the error in the `Action`.
Example:
```
_, _ := runtime.GetRunner().SudoCmd("xxxx")
```

Reply:
ok,deleted IgnoreError

Reply:
ok,I deleted this field